ATAPHtmlReport/CIS_Microsoft_Windows_10_Enterprise_Release_21H1_Benchmark_v1-MITRE ATT&CK Mappings.json

```diff
@@ -0,0 +1,10574 @@
+[
+    {
+        "MITRE ATT\u0026CK Technique 1":  null,
+        "Recommendation #":  null,
+        "Section #":  "1",
+        "Profile":  null,
+        "Title of Recommendation":  "Account Policies",
+        "MITRE ATT\u0026CK Mitigation 2":  null,
+        "MITRE ATT\u0026CK Tactic 2":  null,
```

Comment:
die informationen zur taktik hatten wir gesagt nehmen wir nicht aus der excel da die informationen da teilweise falsch sind.  deswegen gibt es die funktion get-mitreTactic

Reply:
Ist jetzt nicht mehr in der JSON-Datei drin